src/_base/harness/config/pipeline.yml

```diff
@@ -7,15 +7,15 @@ command('app build'): |
 
 command('app build console'): |
   #!bash(workspace:/)|@
-  passthru docker build -t @('docker.repository'):@('app.version')-console -f .my127ws/docker/image/console/Dockerfile .
+  passthru docker build --pull -t @('docker.repository'):@('app.version')-console -f .my127ws/docker/image/console/Dockerfile .
 
 command('app build php-fpm'): |
   #!bash(harness:/docker/image/php-fpm)|@
-  passthru docker build -t @('docker.repository'):@('app.version')-php-fpm .
+  passthru docker build --pull -t @('docker.repository'):@('app.version')-php-fpm .
```

Comment:
going to remove the pull here, in static builds there is a reference to the console image which only exists locally and is not part of a registry. as the console image has been pulled the images needed here should already be up to date.

Reply:
Okay :)